sys/cddl/contrib/opensolaris/uts/common/fs/zfs/dsl_scan.c

```diff
@@ -1,4 +1,4 @@
-/*
+4/*
```

Comment:
Oops.

Reply:
Yeah found that by doing a build AFTER pushing.  Not doing that again!  (My control key is being wonky, as are a couple other keys.)  I've now built world & kernel and pushed it.